public_html/assets/js/nf-core-schema-builder.js

```diff
@@ -539,8 +539,20 @@ $(function () {
         // Populate the help text modal
         var id = $(this).closest('.schema_row').data('id');
         var param = find_param_in_schema(id);
-        $('#help_text_modal .modal-title').html('params.<span>'+id+'</span>');
-        $('.helptext-preview-title').text('--'+id);
+        var modal_header = 'params.<span>'+id+'</span>';
+        var preview_cli_title = '--'+id;
+        var preview_web_title = '<code>--'+id+'</code>';
+        if(param['type'] == 'object'){
+            modal_header = '<span>'+id+'</span>';
+            preview_cli_title = '== '+id+' ==';
+            preview_web_title = id;
+        }
+        if(param['fa_icon'] !== undefined && param['fa_icon'].length > 3){
```

Comment:
Just out of curiosity, why  `param['fa_icon'].length > 3`, is it to check if it is more than `far/fab/fas`?

Reply:
Microsecond random decision as I was typing, so not a lot of thought went into it. But yes, basically.